### PR TITLE
feat(Box): remove className usages

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-2072)

Verified that no tests in webapp are failing after these changes :heavy_check_mark:

### CODE REVIEW

**Review by atabel — APPROVED** [review]

File: src/card.tsx [hunk 1]
```diff
@@ -596,19 +596,15 @@ export const MediaCard = React.forwardRef<HTMLDivElement, MediaCardProps>(
                                 />
                             </div>
                             {asset && (
-                                <Box
-                                    className={styles.mediaCardAsset}
-                                    paddingLeft={{mobile: 16, desktop: 24}}
-                                    paddingTop={{mobile: 16, desktop: 24}}
-                                >
+                                <div className={styles.mediaCardAsset}>
                                     <div
```
Comment (pladaria) [inline src/card.tsx:600]: perhaps these 2 nested divs could be implemented as:
```jsx
<Box paddingX paddingY>
    <div style={applyCssVars(...)} className={styles.mediaCardAsset}>
        ...
    </div>
</Box>
```

Comment (pladaria) [reply src/card.tsx:600]: or even a single one

**Review by pladaria — APPROVED** [review]

File: src/card.tsx (hunk 1 above)
Comment (marcoskolodny) [reply src/card.tsx]: I like the single div option, done

**Pull request merged**